test/kubeclient/kubeclient.go

```diff
@@ -123,6 +131,75 @@ func new(configBytes []byte, namespace string) (*KubeClient, error) {
 	}, nil
 }
 
+func (kc *KubeClient) CreateAzureCredentialsKubeSecret(ctx context.Context) error {
+	serializer := yaml.NewDecodingSerializer(unstructured.UnstructuredJSONScheme)
+	yamlFile, err := os.ReadFile("./config/dev/azure-credentials.yaml")
```

Comment:
Instead of calling `os.ReadFile` on a path that might change use `go:embed` to embed the file at runtime, see: https://gobyexample.com/embed-directive you can embed directly into `[]byte` and then manipulate it as needed below.